xml/installation-installation-configure_lbaas.xml

```diff
@@ -724,338 +606,413 @@ rtt min/avg/max/mdev = 3.473/3.473/3.473/0.000 ms</screen>
-| id                                   | name     | protocol | admin_state_up |
-+--------------------------------------+----------+----------+----------------+
-| 0f5951ee-c2a0-4e62-ae44-e1491a8988e1 | lb1_pool | HTTP     | True           |
-+--------------------------------------+----------+----------+----------------+</screen>
+&prompt.ardana;openstack loadbalancer listener show lb2_listener  --fit-width
++---------------------------+--------------------------------------+
+| Field                     | Value                                |
++---------------------------+--------------------------------------+
+| admin_state_up            | True                                 |
+| connection_limit          | -1                                   |
+| created_at                | 2019-10-29T16:32:26                  |
+| default_pool_id           | 75cd42fa-0525-421f-afaa-5de996267536 |
+| default_tls_container_ref | None                                 |
+| description               |                                      |
+| id                        | 65930b35-70bf-47d2-a135-aff49c219222 |
+| insert_headers            | None                                 |
+| l7policies                |                                      |
+| loadbalancers             | 2b0660e1-2901-41ea-93d8-d1fa590b9cfd |
+| name                      | lb2_listener                         |
+| operating_status          | ONLINE                               |
+| project_id                | de095070f242416cb3dc4cd00e3c79f7     |
+| protocol                  | HTTP                                 |
+| protocol_port             | 80                                   |
+| provisioning_status       | ACTIVE                               |
+| sni_container_refs        | []                                   |
+| timeout_client_data       | 50000                                |
+| timeout_member_connect    | 5000                                 |
+| timeout_member_data       | 50000                                |
+| timeout_tcp_inspect       | 0                                    |
+| updated_at                | 2019-10-29T16:40:01                  |
++---------------------------+--------------------------------------+</screen>
    </step>
   </procedure>
  </section>
  <section>
   <title>Create Floating IPs for Load Balancer</title>
   <para>
-   To create the floating IP's for the load balancer, you will need to list the
-   current ports to get the load balancer id. Once you have the id, you can
-   then create the floating IP.
+   To create the floating IP's for the load balancer, you will need to list the ports for 
+   the load balancer. Once you have the port id, you can then create the floating IP. 
+   Notice that the name has changed between the neutron lbass-loadbalancer and openstack 
+   loadbalancer CLI. In this case be sure to pick the port associated with the name 
+   "octavia-lb-" or use the port id that was saved when the loadbalancer was created.
```

Comment:
```suggestion
   <literal>octavia-lb-</literal> or use the port ID that was saved when the loadbalancer was created.

Reply:
_loadbalancer_ or _load balancer_? It needs to be consistent.

Reply:
looking at https://docs.openstack.org/octavia/latest/reference/introduction.html  it looks like "load balancer" when its not explicitly being used in the command-line, in which case its loadbalancer .... I'll update